tests/e2e/helpers/keys.ts

```diff
@@ -214,3 +214,13 @@ export async function deleteAllKeysFromDB(host: string, port: string): Promise<v
         await client.quit();
     });
 }
+
+/**
+* Verifying if the Keys are in the List of keys
+* @param keyNames The names of the keys
+*/
+export async function verifyKeysDisplayedInTheList(keyNames: string[]): Promise<void> {
+    for (const keyName of keyNames) {
+        await t.expect(await browserPage.isKeyIsDisplayedInTheList(keyName)).ok(`The key ${keyName} not found`);
```

Comment:
Consider using `.visible`

Reply:
there is .exists check into the isKeyIsDisplayedInTheList() method
![image](https://user-images.githubusercontent.com/90257586/199705314-8c1dd931-01cd-4334-b7c7-8000b6d6d2ff.png)